SofaKernel/framework/sofa/helper/io/MassSpringLoader.cpp

```diff
@@ -105,6 +105,7 @@ bool MassSpringLoader::load(const char *filename)
                 msg_error("MassSpringLoader") << "fscanf function has encountered an error." ;
                 setNumMasses(0);
                 setNumSprings(0);
+                fclose(file);
```

Comment:
Isn't there RAII pattern to handle this kind of situation ?

Reply:
See https://en.cppreference.com/w/cpp/io/basic_ofstream
and https://eli.thegreenplace.net/2016/c-raii-without-exceptions/